espresso/core/java/androidx/test/espresso/matcher/ViewMatchers.java

```diff
@@ -1381,7 +1387,8 @@ protected void describeMoreTo(Description description) {
         description.appendText(" [").appendText(resourceName).appendText("]");
       }
       if (null != expectedText) {
-        description.appendText(" value: ").appendText(expectedText);
+        description.appendText(" value: ");
+        SpecialCharVisualizer.appendHighlightedValue(expectedText, description, true);
```

Comment:
It doesn't seem this SpecialCharVisualizer#appendHighlightedValue provides much value. I'd suggest removing the method. You can simply call `description.appendText(" value: ").appendText(highlight(expectedText))`

Reply:
`appendHighlightedValue` provides highlighting plus the hint text for better comprehension.
Previously, we shared the idea that this should be done to exclude confusion while perceiving the raw value of the string.

Reply:
I'm still not convinced that this `appendHighlightedValue` wrapper method provides much value. Can we just keep separate methods when we handle value or text?